packages/animations/test/open_container_test.dart

```diff
@@ -1733,6 +1733,46 @@ void main() {
       expect(modalRoute.settings, routeSettings);
     },
   );
+
+  testWidgets(
+    'OpenContainer works when the route is removed',
```

Comment:
Maybe:
```suggestion
    'OpenContainer's source widget is visible in closed container route if open container '
    'route is pushed from not using the OpenContainer itself',
```

I'm trying to make this description more helpful, but I think that the link to the issue does help enough